test/OpenTelemetry.Contrib.Instrumentation.Quartz.Tests/OpenTelemetry.Contrib.Instrumentation.Quartz.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <Description>Unit test project for OpenTelemetry Quartz.NET instrumentation</Description>
+        <TargetFrameworks>netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
```

Comment:
we can remove netcoreapp2.1, as its EOL very soon! (the main repo is in the process of removing it already).

Also, do you want to add atleast one .NET Framework target for the tests/

Reply:
removed netcoreapp2.1 and added net472